### PR TITLE
stats_monitor: reformat stats_monitor tick

### PR DESCRIPTION
**Pull request by lhy1024**
Signed-off-by: lhy1024 <admin@liudos.us>

### What is changed and how it works?

- rename `collect_interval` to `collect_tick_interval`
- rename `qps_info_interval` to `load_base_split_check_interval`
- rename `thread_info_interval` to `collect_store_infos_interval`
- add two function 

### Check List 

Tests 

- Unit test

### Release note 

```release-note
None
```

### CODE REVIEW

File: components/raftstore/src/store/worker/pd.rs [hunk 1]
```diff
@@ -400,22 +402,21 @@ where
...
-            .div_duration_f64(self.collect_interval) as i32;
+        let tick_interval = self.collect_tick_interval;
+        let collect_store_infos_interval = self
+            .collect_store_infos_interval
+            .div_duration_f64(tick_interval) as i32;
+        let load_base_split_interval = self
+            .load_base_split_interval
+            .div_duration_f64(tick_interval) as i32;
+
         let (tx, rx) = mpsc::channel();
```
[the remark this answers is not in the document]
Comment (lhy1024) [reply components/raftstore/src/store/worker/pd.rs:420]: Do you have any suggestions for their names?

[the remark this answers is not in the document]
Comment (JmPotato) [reply components/raftstore/src/store/worker/pd.rs:420]: What about:

* `self.timer` -> `self.collect_interval_timer`
* `self.sender` -> `self.read_stats_sender`

[the remark this answers is not in the document]
Comment (lhy1024) [reply components/raftstore/src/store/worker/pd.rs:420]: PTAL

File: components/raftstore/src/store/worker/pd.rs [hunk 2]
```diff
@@ -342,13 +342,15 @@ where
     }
 }
 
-const DEFAULT_QPS_INFO_INTERVAL: Duration = Duration::from_secs(1);
-const DEFAULT_COLLECT_INTERVAL: Duration = Duration::from_secs(1);
+const DEFAULT_LOAD_BASE_SPLIT_INTERVAL: Duration = Duration::from_secs(1);
```
Comment (JmPotato) [inline components/raftstore/src/store/worker/pd.rs:345]: I suggest using `DEFAULT_LOAD_BASE_SPLIT_CHECK_INTERVAL`.

File: components/raftstore/src/store/worker/pd.rs [hunk 3]
```diff
@@ -386,11 +388,11 @@ where
...
             handle: None,
-            timer: None,
-            sender: None,
-            thread_info_interval: interval,
-            qps_info_interval: cmp::min(DEFAULT_QPS_INFO_INTERVAL, interval),
-            collect_interval: cmp::min(DEFAULT_COLLECT_INTERVAL, interval),
+            ticker_timer: None,
+            read_stats_sender: None,
+            collect_store_infos_interval: interval,
+            load_base_split_interval: cmp::min(DEFAULT_LOAD_BASE_SPLIT_INTERVAL, interval),
```
Comment (JmPotato) [inline components/raftstore/src/store/worker/pd.rs:394]: Ditto.

File: components/raftstore/src/store/worker/pd.rs [hunk 4]
```diff
@@ -400,27 +402,26 @@ where
...
-            .thread_info_interval
-            .div_duration_f64(self.collect_interval) as i32;
-        let qps_info_interval = self
-            .qps_info_interval
-            .div_duration_f64(self.collect_interval) as i32;
+        let tick_interval = self.collect_tick_interval;
+        let collect_store_infos_interval = self
+            .collect_store_infos_interval
+            .div_duration_f64(tick_interval) as i32;
+        let load_base_split_interval = self
```
Comment (JmPotato) [inline components/raftstore/src/store/worker/pd.rs:416]: Ditto.

**Review by JmPotato — APPROVED** [review]
The rest LGTM.

File: components/raftstore/src/store/worker/pd.rs [hunk 5]
```diff
@@ -370,11 +372,11 @@ where
 {
     scheduler: Scheduler<Task<EK, ER>>,
     handle: Option<JoinHandle<()>>,
-    timer: Option<Sender<bool>>,
-    sender: Option<Sender<ReadStats>>,
-    thread_info_interval: Duration,
-    qps_info_interval: Duration,
-    collect_interval: Duration,
+    ticker_timer: Option<Sender<bool>>,
```
Comment (rleungx) [inline components/raftstore/src/store/worker/pd.rs:375]: IMP, `ticker_timer` is not a good naming.

File: components/raftstore/src/store/worker/pd.rs [hunk 6]
```diff
@@ -400,27 +402,26 @@ where
         &mut self,
         mut auto_split_controller: AutoSplitController,
     ) -> Result<(), io::Error> {
-        if self.collect_interval < default_collect_interval() {
+        if self.collect_tick_interval < default_collect_tick_interval()
+            || self.collect_store_infos_interval < self.collect_tick_interval
+        {
             info!("it seems we are running tests, skip stats monitoring.");
```
Comment (rleungx) [inline components/raftstore/src/store/worker/pd.rs:408]: `it seems we are running tests` looks wired if it happens in real production.

**Review by rleungx — APPROVED** [review]

**Review by AndreMouche — APPROVED** [review]
LGTM

**Pull request merged**